web/static/css/layout.scss

```diff
@@ -11,3 +11,9 @@
   flex: 1;
 }
 
+#main-nav {
```

Comment:
Avoid using id selectors

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#idselector)